syft/frameworks/crypten/context.py

```diff
@@ -26,6 +26,7 @@ def _launch(func, rank, world_size, master_addr, master_port, queue, func_args,
         os.environ[key] = str(val)
```

Comment:
Some of the changes from this file are taken from [here](https://github.com/OpenMined/PySyft/pull/3565)